src/config/artifacts.rs

```diff
@@ -36,6 +38,14 @@ pub struct Artifacts {
     pub package_managers: Option<LinkedHashMap<String, String>>,
 }
 
+impl ApplyLayer for Artifacts {
+    fn apply_layer(&mut self, layer: Self) {
+        self.cargo_dist.apply_opt(layer.cargo_dist);
+        // FIXME: should this get merged with e.g. `extend?`
```

Comment:
where is the other location that a user can specify package_managers?

Reply:
Nowhere, but each layer """has to""" assume that other layers can set these values, as a matter of like, robust abstraction?

Reply:
These FIXMEs are more like "hey this code might make your sad if we add new layers that touch these values".